src/core/getters/combine.mock.ts

```diff
@@ -55,9 +55,9 @@ export const combineSchemasMock = async ({
       if (!index && !combine) {
         if (resolvedValue.enums || isOneOf) {
           if (arr.length === 1) {
-            return `faker.helpers.randomize([${resolvedValue.value}])`;
+            return `faker.random.arrayElement([${resolvedValue.value}])`;
```

Comment:
That's a good question. I didn't write the original code so for the sake of keeping the PR as small as possible I decided not to touch the logic in case I misunderstood its purpose, and simply replace the deprecated function with its recommended replacement.

I could remove the call to faker altogether instead if that is preferred.

Reply:
it's a list of enums, there is maybe only one enum in the list but an enum can have multiple values

Reply:
@yekver is it ok for you?

Reply:
> @yekver is it ok for you?

According to the scope of current PR and it's purpose - yes, it's ok for me.

P.S.
I was confused by `arr.length === 1` and didn't realize that `resolvedValue.value` could be an `array`.
I think this part might be refactored a bit in future.